src/Microsoft.Azure.SignalR/ServerConnections/ServiceConnection.cs

```diff
@@ -136,10 +138,7 @@ protected override Task OnClientDisconnectedAsync(CloseConnectionMessage closeCo
             var connectionId = closeConnectionMessage.ConnectionId;
             if (_enableConnectionMigration && _clientConnectionManager.ClientConnections.TryGetValue(connectionId, out var context))
             {
-                if (!context.HttpContext.Request.Headers.ContainsKey(Constants.AsrsMigrateOut))
-                {
-                    context.HttpContext.Request.Headers.Add(Constants.AsrsMigrateOut, "");
-                }
+                context.Features.Set<IConnectionMigrationFeature>(new ConnectionMigrationFeature("", ""));

```

Comment:
what if this connection is migrated in? this one will override the previous one?